

try:
    s = int(input("Enter your no: "))
    print(s+4)
except Exception as e:
    print(e)
    print("some invaild input.Please enter again")
    
    
    
    